source/portable/NetworkInterface/ATSAME5x/NetworkInterface.c

```diff
@@ -81,7 +81,7 @@
 
 /* Setup LLMNR specific multicast address. */
 #if ( defined( ipconfigUSE_LLMNR ) && ( ipconfigUSE_LLMNR == 1 ) )
-    static const uint8_t ucLLMNR_MAC_address[] = { 0x01, 0x00, 0x5E, 0x00, 0x00, 0xFC };
+    static uint8_t ucLLMNR_MAC_address[] = { 0x01, 0x00, 0x5E, 0x00, 0x00, 0xFC };
```

Comment:
We have this definition in FreeRTOS_DNS.h.
> /* extern const MACAddress_t xLLMNR_MacAdress;

Is it possible to re-use that?

Reply:
The MAC driver API that is used to set the MAC filter (`mac_async_set_filter_ex`) expects a non const pointer to the MAC address hence we wont be able to use the const definition of the LLMNR MAC address from the FreeRTOS_DNS.h.

Reply:
How about using a pointer?
```
uint8_t * pucLLMNR_MAC_address = &( xLLMNR_MacAdress.ucBytes[0] );
```